drivers/display/ssd16xx.c

```diff
@@ -623,8 +681,8 @@ static int ssd16xx_clear_cntlr_mem(const struct device *dev, uint8_t ram_cmd)
 	 * Clear unusable memory area when the resolution of the panel is not
 	 * multiple of an octet.
 	 */
-	if (config->height % EPD_PANEL_NUMOF_ROWS_PER_PAGE) {
-		panel_h += 1;
+	if (config->quirks->x_addr_in_bytes) { 
+		panel_h = (config->height / EPD_PANEL_NUMOF_ROWS_PER_PAGE) + (config->height % EPD_PANEL_NUMOF_ROWS_PER_PAGE ? 1 : 0);
```

Comment:
Why make a ternary statement here? We can just round the division up, right?

`panel_h = (config->height + (EPD_PANEL_NUMOF_ROWS_PER_PAGE - 1)) / EPD_PANEL_NUMOF_ROWS_PER_PAGE;`

Reply:
Yes, we should be able to do that. I just kept it similar to what we were doing before. Will change.

Reply:
Updated